assets/app.js

```diff
@@ -1,16 +1,16 @@
 $(function() {
   // Constants
   const DESKTOP_WIDTH = 639;
-  const BACKGROUND_COLORS = [
-    '#FFEB82',
-    '#FAAF73',
-    '#F77787',
-    '#FAAFD7',
-    '#BE96FF',
-    '#7D9BE1',
-    '#96E1FA',
-    '#A0F0C3'
-  ];
+  // const BACKGROUND_COLORS = [
```

Comment:
Same as above, let’s remove if it’s not needed, and if we ever want it back we can find it in git history